desktop/src/main/fileProcessing/workers/utilities/countFiles.ts

```diff
@@ -17,5 +18,13 @@ export const countFiles = async (dir: string): Promise<number> => {
     }
   }
 
+  if (count === 0) {
+    // Folder is empty
+    parentPort?.postMessage({
+      type: "emptyFolder",
+      path: dir,
+    });
```

Comment:
sending a msg here makes more sense to me. (See duplicated msg sending in `copyWorker.ts`)

Reply:
Its actually not using that utility function. Scott originally only refactored out the metadata worker to use the utility functions and not the copy worker. Ill do that for the copy worker.